{"name": "test_stuff", "status": "failed", "statusDetails": {"message": "AssertionError: assert 5 == 4", "trace": "@pytest.mark.basic\n    def test_stuff():\n>       assert 5 ==4\nE       assert 5 == 4\n\ntest_basic.py:5: AssertionError"}, "start": 1678798521088, "stop": 1678798521088, "uuid": "434fd9fe-69bc-4977-b56d-44950238bce2", "historyId": "d2a1f0c5a0336d6f814f11a788dd8ce7", "testCaseId": "67140f0addf832956f41477533c68b39", "fullName": "test_basic#test_stuff", "labels": [{"name": "tag", "value": "basic"}, {"name": "suite", "value": "test_basic"}, {"name": "host", "value": "test-users-air.ntdom.local"}, {"name": "thread", "value": "55218-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_basic"}]}